.github/workflows/time-tracker-ui-ci.yml

```diff
@@ -0,0 +1,146 @@
+name: time-tracker-ui-ci
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    env:
+        WORKING_DIR: infrastructure/
+        DB_CONNECTION:  ${{ secrets.DB_CONNECTION }}
+        ARM_CLIENT_ID: ${{secrets.TF_ARM_CLIENT_ID}}
+        ARM_CLIENT_SECRET: ${{secrets.TF_ARM_CLIENT_SECRET}}
+        ARM_SUBSCRIPTION_ID: ${{secrets.TF_ARM_SUBSCRIPTION_ID}}
+        ARM_TENANT_ID: ${{secrets.TF_ARM_TENANT_ID}}
+    strategy:
+      max-parallel: 5
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Authenticate with the TF modules repository
+      uses: webfactory/ssh-agent@v0.5.4
+      with:
+        ssh-private-key: ${{ secrets.INFRA_TERRAFORM_MODULES_SSH_PRIV_KEY }}
+
+    #     ci-test:
+    # name: Unit tests and CodeCoverage
+    # timeout-minutes: 90
+    # steps:
+    # - name: Set up Node.js version
+    #   uses: actions/setup-node@v1
+    #   with:
+    #     node-version: '12.x'
+
+    # - name: Install dependencies
+    #   run: npm install
+
+    - name: build docker
+      run: make build
+
+    - name: Inject Secrets
+      env:
+        SCOPES: ${{ secrets.SCOPES }}
+        CLIENT_ID: ${{ secrets.CLIENT_ID }}
+        AUTHORITY: ${{ secrets.AUTHORITY }}
+        STACK_EXCHANGE_ID: ${{ secrets.STACK_EXCHANGE_ID }}
+        STACK_EXCHANGE_ACCESS_TOKEN: ${{ secrets.STACK_EXCHANGE_ACCESS_TOKEN }}
+        AZURE_APP_CONFIGURATION_CONNECTION_STRING: ${{ secrets.AZURE_APP_CONFIGURATION_CONNECTION_STRING }}
+      run: |
+        chmod +x ./scripts/populate-keys.sh
+        sh ./scripts/populate-keys.sh
+
+    - name: Running tests
+      run: |
+        chmod -R 777 ./$home
+        make test
+    - name: Generate coverage report
+      env:
+        CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+      run: bash <(curl -s https://codecov.io/bash)
+
+    - name: Setup terraform
+      uses: hashicorp/setup-terraform@v1
+
+    - name: 'Terraform Init'
+      id: init
+      working-directory: ./${{ env.WORKING_DIR }}
+      run: terraform init
+
+    - name: 'Terraform validate'
+      id: validate
+      working-directory: ./${{ env.WORKING_DIR }}
+      run: terraform validate
+
+    - name: Terraform Plan Stage
+      id: plan-stage
+      if: github.event_name == 'pull_request'
+      run: terraform plan -var-file=${{ env.TF_WORKSPACE }}.tfvars -var image_tag=latest -no-color
+      continue-on-error: true
```

Comment:
why is this continue-on-error set to true? Should we allow the job to pass even if this step of Terraform Plan Stage fails?

Reply:
yes, there are steps that runs in case of a plan error